Content.Server/Labels/Label/Components/LabelComponent.cs

```diff
@@ -1,12 +1,22 @@
 namespace Content.Server.Labels.Components
 {
+    /// <summary>
+    /// Gives opportunity to add labels by entities with the <see cref="HandLabelerComponent"/>
+    /// </summary>
     [RegisterComponent]
     public sealed class LabelComponent : Component
     {
+        /// <summary>
+        /// Label text. If you using that field in your prototype, you must set originalName field too, and set name of entity like it with label
+        /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("currentLabel")]
         public string? CurrentLabel { get; set; }
 
+        /// <summary>
+        /// Name of entity without label
+        /// </summary>
+        [DataField("originalName")]
```

Comment:
I dont think this is a good solution, the current problem with the jugs is that the label is in their names, which bypasses the component. I think a better fix would be reversing these changes and replacing their name label with a suffix, for example
```yml
  name: jug (Carbon)
  id: JugCarbon
```
  would become
```yml
  name: jug
  suffix: Carbon
  id: JugCarbon
```
 but do tell if it doesnt work

Reply:
suffix isnt shown in vendors???

Reply:
dont they display labels as well? seems like something to fix if they dont

Reply:
Maybe remove originalName field and rework all systems that need to show labels (examine, handLabeler, vendomates and something else). The main idea that if you want use labels, then you use it. 

Or add to MeatadataComponent nameToShow field, that must be without DataField atribute. That field required for constructing name that was be showen to user by examine and other, but vendomates taking info from prototypes, this was not fix it. Also then i must recode systems that must use labels.

I just want to know is this a good idea?

Reply:
I think originalName will always be necessary to store, tho if you find a solution without it, thats better.  Modifying MetadataComponent is also not a very good idea. I think just making vendors show their label and using the suffix like i suggested is the easiest thing to do for now